src/thing_creature.c

```diff
@@ -3433,9 +3432,16 @@ TbBool thing_is_creature_special_digger(const struct Thing *thing)
   return ((get_creature_model_flags(thing) & CMF_IsSpecDigger) != 0);
 }
 
-void anger_set_creature_anger_all_types(struct Thing *thing, long a2)
+void anger_set_creature_anger_all_types(struct Thing *thing, long new_value)
 {
-    _DK_anger_set_creature_anger_all_types(thing, a2);
+    // _DK_anger_set_creature_anger_all_types(thing, a2);
```

Comment:
still think lines like these are just garbage that'll need to be cleaned up later anyways, if one wanted to go back to an older version of a function using git history should really be the way to go